Makefile

```diff
@@ -49,8 +49,8 @@ SERIAL_DEVICE   ?= $(firstword $(wildcard /dev/ttyACM*) $(firstword $(wildcard /
 # Flash size (KB).  Some low-end chips actually have more flash than advertised, use this to override.
 FLASH_SIZE ?=
 
-# Release file naming (no revision to be present if this is 'yes')
-RELEASE ?= no
+# Simple file naming (no revision to be present if this is 'no')
+REVISION ?= 
```

Comment:
Can we have the default to be `no` so that the previous behavior of the build system is restored which will allow it once again to work with scripts that require the artifacts to always have the same names?

Reply:
This would be reverting to behaviour that has long since not been in use.

Reply:
yes, I understand that. which is why my original proposal in #11659 seeks add the additional step, thus satisfying the CURRENT requirements of having builds that default to having the revision number added *and* also the PREVIOUS behavior which generate the files that scripts and workflows, especially ones using external flash chips, use.

The only issue with #11659 was the Jenkins job, no other solution proposed satisfies the current and previous requirements.

Can we please get the jenkins job fixed and revert #11687?

Reply:
I actually don't think the Jenkins job was the problem. The behaviour for `make all` was by default to add the revision. Your change altered that behaviour. Simply put you broke an expected behaviour, without considering the consequences.

The question is - do we want that behaviour to continue in any case. i.e. is there any actual value in adding the revision to the filename, when the revision is added into the firmware output in cli anyway. What purpose does it serve other than if you want a number of different builds for distribution for testing, in which case adding REVISION=Yes will now give you that outcome.